server/db/db.js

```diff
@@ -1,2 +1,10 @@
-//create db
-//export db
+const Sequelize = require('sequelize')
+
+const conn = new Sequelize(
+    process.env.DATABASE_URL || 'postgres://localhost:5432/snacker',
+    {
+        logging: false,
+    }
+)
+
+module.exports = conn
```

Comment:
You are importing db throughout your other files but export conn here. Not sure if the naming could cause issues here.